<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkPreprocessingResamplingViewControls</class>
 <widget class="QWidget" name="QmitkPreprocessingResamplingViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="3" column="0">
    <widget class="QCommandLinkButton" name="buttonExecuteOnMultipleImages">
     <property name="text">
      <string>Resample all selected images</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="leImage1">
        <property name="text">
         <string>Select an image in the Data Manager</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tlTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Output image will be 3D</string>
        </property>
        <property name="text">
         <string>Choose time step if 4D
(Slider for both images)</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QmitkSliderNavigatorWidget" name="sliceNavigatorTime" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Output image will be 3D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>254</width>
       <height>403</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QCommandLinkButton" name="btnDoIt">
     <property name="text">
      <string>Resample single image</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbOneImageOps">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dsbParam3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dsbParam2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="dsbParam1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="tlParam">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Resampling Parameter</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tlParam1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>x-spacing</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="tlParam2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>y-spacing</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="cbHideOrig">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Hide Original Image</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="tlParam3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>z-spacing</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="tlParam4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Interpolation:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cbParam4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkSliderNavigatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSliderNavigatorWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dsbParam1</tabstop>
  <tabstop>dsbParam2</tabstop>
  <tabstop>dsbParam3</tabstop>
  <tabstop>cbParam4</tabstop>
  <tabstop>cbHideOrig</tabstop>
  <tabstop>btnDoIt</tabstop>
  <tabstop>buttonExecuteOnMultipleImages</tabstop>
  <tabstop>leImage1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
